.gitignore

```diff
@@ -32,4 +32,5 @@ CMakeLists.txt.user
 
 # Other Stuff
 tags
-*.d
\ No newline at end of file
+*.d
+_build/
```

Comment:
This seems a change for your local setup. Please exclude it from the PR!

Reply:
Thanks for pointing this out. I've resolved it now :)

Reply:
Excellent!